src/Object/Post.php

```diff
@@ -1068,7 +1068,9 @@ private function getCommentBox(string $indent)
 				'$prompttext'  => DI::l10n()->t('Please enter a image/video/audio/webpage URL:'),
 				'$preview'     => DI::l10n()->t('Preview'),
 				'$indent'      => $indent,
-				'$rand_num'    => Crypto::randomDigits(12)
+				'$rand_num'    => Crypto::randomDigits(12),
+				// Dropzone
+				'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize')) / 1000000,1),
```

Comment:
Code standards: Please add a space after commas.
```suggestion
				'$max_imagesize' => round(\Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize')) / 1000000, 1),
```